apps/client/src/app/core/simulation/simulation.service.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from '@angular/core';
+import * as Simulator from '@ffxiv-teamcraft/simulator';
+import { Region } from '../../modules/settings/region.enum';
+import { getSimulator } from './simulation';
+
+export type ActionResult = Simulator.ActionResult;
+export type ActionType = Simulator.ActionType;
+export type Craft = Simulator.Craft;
+export type CraftingJob = Simulator.CraftingJob;
+export type CrafterLevels = Simulator.CrafterLevels;
+export type CrafterStats = Simulator.CrafterStats;
+export type CraftingAction = Simulator.CraftingAction;
+export type EffectiveBuff = Simulator.EffectiveBuff;
+export type GearSet = Simulator.GearSet;
+export type Simulation = Simulator.Simulation;
+export type SimulationReliabilityReport = Simulator.SimulationReliabilityReport;
+export type SimulationResult = Simulator.SimulationResult;
+export type StepState = Simulator.StepState;
```

Comment:
Are these still needed? I don't see them inside the other changed files.

Reply:
https://github.com/ffxiv-teamcraft/ffxiv-teamcraft/pull/1373/files#diff-13914238e5098788f9e876d127449e86R59 Here are them imported